libnopegl/src/pass.c

```diff
@@ -165,6 +165,14 @@ static int register_texture(struct pass *s, const char *name, struct ngl_node *t
                     LOG(ERROR, "\"%s\" can not be accessed as an image; only Texture{2D,2DArray,3D,Cube} are supported", name);
                     return NGL_ERROR_UNSUPPORTED;
                 }
+
+                const struct ngl_ctx *ctx = s->ctx;
+                const struct gpu_ctx *gpu_ctx = ctx->gpu_ctx;
+                if (!(gpu_ctx->features & NGLI_FEATURE_COMPUTE)) {
+                    LOG(ERROR, "context does not support image load store operations");
+                    return NGL_ERROR_GRAPHICS_UNSUPPORTED;
+                }
```

Comment:
We can't have read only images without compute?

BTW I think both checks belong in the `init` callback of the `ResourcesProps` node

Reply:
Done. Also added NGLI_FEATURE_IMAGE_LOAD_STORE in a separate commit.